slack/apps-and-integrations/apps-and-integrations.md

```diff
@@ -0,0 +1,12 @@
+# List of apps and integrations used
+
+## Productboard
+- **Description:** Productboard is a product management tool, used by the PrestaShop Core's product team to gather feedback, organize and pritoritize features and improvements to build, and to follow their development.
+The Productboard app connects this Slack to the PrestaShop's Productboard.  
+It allows the product managers to transform a Slack message (and its comments) into a feedback "note" on productbaord.  
+More info on the productboard website: https://www.productboard.com/
+- **Access:** Limited
```

Comment:
can you please explain that there is also a public view and add the link?

Reply:
added the link to the PB Portal (http://feedback.prestashop.com/) though, people won't see their feedback there, they will only see the ideas that result from their feedback after we grouped them and organize them